mne/gui/_file_traits.py

```diff
@@ -321,20 +327,28 @@ def _get_points(self):
             return self.inst_points[self.points_filter]
 
     @cached_property
-    def _get_fid_dig(self):  # noqa: D401
-        """Fiducials for info['dig']."""
+    def _get_fid_dig(self):
+        """Get fiducials from info['dig']."""
         if not self.inst:
             return []
-        dig = self.inst['dig']
-        dig = [d for d in dig if d['kind'] == FIFF.FIFFV_POINT_CARDINAL]
+        dig = [d for d in self.inst['dig']
+               if d['kind'] == FIFF.FIFFV_POINT_CARDINAL]
+        return dig
+
+    @cached_property
+    def _get_eeg_dig(self):
+        """Get EEG from info['dig']."""
+        if not self.inst:
+            return np.empty((0, 3))
```

Comment:
I actually think this is more consistent. You always get a 2D array with shape[-1] == 3 this way. I've hit errors before where someone returns a list or `np.array([])` instead of `np.empty((0, 3))` or at least `np.array([[]])`... some functions can (reasonably) assume consistent returns